docs/docs/developers/tutorials/crowdfunding/donations.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Build a donations contract
+tags: [developers, tutorial, example]
+---
+
+In this tutorial we'll create two contracts related to crowdfunding:
+
+- A crowdfunding contract with two core components
+  - Fully private donations
+  - Verifiable withdrawals to the operator
+- A reward contract for anyone else to anonymously reward donors
+
+Along the way you will:
+
+- Install Aztec developer tools
+- Setup a new Noir contract project
+- Add base Aztec dependencies
+- 
+
+## Setup
+
+### Install tools
+
+Please ensure that the you already have [Installed the Sandbox](https://docs.aztec.network/developers/getting_started/quickstart#install-the-sandbox).
+
+And if using VSCode, see [here](https://docs.aztec.network/developers/contracts/main#install-noir-lsp-recommended) to install Noir LSP, where you'll benefit from syntax highlighting, profiling, and more.
+
+### Create an Aztec project
+
+Use `aztec-nargo` in a terminal to create a new Aztec contract project named "crowdfunding":
+
+```sh
+aztec-nargo new --contract crowdfunding
+```
+
+Inside the new `crowdfunding` directory you will have a base to implement the Aztec smart contract.
+
+Use `aztec-nargo --help` to see other commands.
+
+## Private donations
+
+1. An "Operator" begins a Crowdfunding campaign (contract), specifying:
+
+- an existing token address
+- their account address
+- a deadline timestamp
+
+2. Any address can donate (in private context)
+
+- private transfer token from sender to contract
+- privately note amount donated (claimable via other contract)
```

Comment:
```suggestion
- transaction receipts allow private claims via another contract
```
Should you add a link to the rewards tutorial here?

Reply:
I link to the rewards contract at the end, and removed the stub from the sidebar (an earlier commit). I'll remove the .md too